Tests/FunctionalTest.php

```diff
@@ -457,7 +457,7 @@ public function testWithIntrospection(): void
 
     public function testDisableIntrospection(): void
     {
-        $kernel = new GraphQLiteTestingKernel(true, null, true, null, false, 2, 2);
+        $kernel = new GraphQLiteTestingKernel(true, null, true, null, false, 3, 2);
```

Comment:
I guess the webonyx pkg changed how they calculate complexity, it was now failing on 2 raised it to 3